Common/Api/OptimizationBacktestJsonConverter.cs

```diff
@@ -0,0 +1,221 @@
+        /// <param name="existingValue">The existing value of object being read.</param>
+        /// <param name="serializer">The calling serializer.</param>
+        /// <returns>
+        /// The object value.
+        /// </returns>
+        public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
+        {
+            var jObject = JObject.Load(reader);
+
+            var optimizationBacktest = CreateOptimizationBacktestFromJObject(jObject, serializer);
+
+            return optimizationBacktest;
+        }
+
+        /// <summary>
+        /// Custom parsing of optimization backtest data
+        /// </summary>
+        /// <param name="jObject">JSON representing OptimizationBacktest.</param>
+        /// <param name="serializer">The calling serializer.</param>
+        /// <returns></returns>
+        public static OptimizationBacktest CreateOptimizationBacktestFromJObject(JObject jObject, JsonSerializer serializer)
+        {
+            var name = jObject["name"].Value<string>();
+            var hostName = jObject["hostName"]?.Value<string>();
+            var backtestId = jObject["id"].Value<string>();
+            var progress = jObject["progress"].Value<decimal>();
+            var exitCode = jObject["exitCode"].Value<int>();
+
+            var jStatistics = jObject["statistics"];
+            var statistics = new Dictionary<string, string>
+            {
+                { "Alpha", jStatistics[0].Value<string>() },
+                { "Annual Standard Deviation", jStatistics[1].Value<string>() },
+                { "Annual Variance", jStatistics[2].Value<string>() },
+                { "Average Loss", jStatistics[3].Value<string>() },
+                { "Average Win", jStatistics[4].Value<string>() },
+                { "Beta", jStatistics[5].Value<string>() },
+                { "Compounding Annual Return", jStatistics[6].Value<string>() },
+                { "Drawdown", jStatistics[7].Value<string>() },
+                { "Estimated Strategy Capacity", jStatistics[8].Value<string>() },
+                { "Expectancy", jStatistics[9].Value<string>() },
+                { "Information Ratio", jStatistics[10].Value<string>() },
+                { "Loss Rate", jStatistics[11].Value<string>() },
+                { "Net Profit", jStatistics[12].Value<string>() },
+                { "Probabilistic Sharpe Ratio", jStatistics[13].Value<string>() },
+                { "Profit-Loss Ratio", jStatistics[14].Value<string>() },
+                { "Sharpe Ratio", jStatistics[15].Value<string>() },
+                { "Total Fees", jStatistics[16].Value<string>() },
+                { "Total Trades", jStatistics[17].Value<string>() },
+                { "Tracking Error", jStatistics[18].Value<string>() },
+                { "Treynor Ratio", jStatistics[19].Value<string>() },
+                { "Win Rate", jStatistics[20].Value<string>() },
```

Comment:
cool! I think we could try to avoid the duplication with `StatisticsBuilder` also defining these string keys maybe they could be defined as a static or cont in some common place like `statistics.cs`

Reply:
Good idea. I've defined the strings in a new PerformanceMetrics class so as not to add more bloat to the old Statistics class.